code/game/objects/items/stacks/sheets/leather.dm

```diff
@@ -206,6 +206,12 @@ GLOBAL_LIST_INIT(leather_recipes, list ( \
 	singular_name = "wolf sinew"
 	merge_type = /obj/item/stack/sheet/sinew/wolf
 
+/obj/item/stack/sinew/attack_secondary(mob/living/victim, mob/living/user, params)
+	if (user == victim)
+		return SECONDARY_ATTACK_CANCEL_ATTACK_CHAIN
+	user.AddComponent(/datum/component/garrote, victim, src, 0.5)
```

Comment:
are components supposed to be used in this way? Seems weird to constantly stack on components that never get removed if you garrote someone multiple times

Reply:
A good example might be the currently fixed gunpointing component for something similar to this.